srcpkgs/dhcp-helper/template

```diff
@@ -0,0 +1,13 @@
+# Template file for 'dhcp-helper'
+pkgname=dhcp-helper
+version=1.2
+revision=1
+_srcver=1
+build_style=gnu-makefile
+make_install_args="BINDIR=/usr/bin MANDIR=/usr/share/man"
+short_desc="Simple, straightforward DHCP relay agent"
+maintainer="fosslinux <fosslinux@aussies.space>"
+license="GPL-2.0-or-later"
```

Comment:
As stated here: https://metadata.ftp-master.debian.org/changelogs//main/d/dhcp-helper/dhcp-helper_1.2-1_copyright

This is `GPL-2.0-only OR GPL-3.0-only`.

Othewise this is fine since the Debian maintainer also the author of this package.
However, use Buster as a homepage is not a good idea, I think.

I guess `sid` or this page `http://www.thekelleys.org.uk/dhcp-helper/` would be better.

Reply:
Yeah, fair enough - went with the upstream distfiles.

Reply:
Should it be `GPL-2.0-only OR GPL-3.0-only` or `GPL-2.0-only, GPL-3.0-only` in the field?